plugins/modules/cloudwatch_metric_alarm_info.py

```diff
@@ -0,0 +1,203 @@
+            returned: always
+            type: str
+        metric_name:
+            description: Name of the monitored metric (e.g. C(CPUUtilization)).
+            returned: always
+            type: str
+        namespace:
+            description:
+                - Name of the appropriate namespace (C(AWS/EC2), C(System/Linux), etc.).
+                - Determines the category it will appear under in CloudWatch.
+            returned: always
+            type: str
+        ok_actions:
+            description: The actions to execute when this alarm transitions to an OK state from any other state.
+            returned: always
+            type: str
+        period:
+            description:
+                - The length, in seconds, used each time the metric specified in MetricName is evaluated.
+                - Valid values are 10, 30, and any multiple of 60.
+            returned: always
+            type: int
+        state_reason:
+            description: An explanation for the alarm state, in text format.
+            returned: always
+            type: str
+        state_reason_data:
+            description: An explanation for the alarm state, in JSON format.
+            returned: always
+            type: str
+        state_updated_timestamp:
+            description: The time stamp of the last update to the alarm state.
+            returned: always
+            type: str
+        state_value:
+            description: The state value for the alarm.
+            returned: always
+            type: str
+        statistic:
+            description: The statistic for the metric associated with the alarm, other than percentile.
+            returned: always
+            type: str
+        threshold:
+            description: The value to compare with the specified statistic.
+            returned: always
+            type: str
+        treat_missing_data:
+            description: Sets how alarm is to handle missing data points.
+            returned: always
+            type: str
+        unit:
+            description: Unit used when storing the metric
+            returned: always
+            type: str
+'''
+
+
+try:
+    import botocore
+except ImportError:
+    pass  # Handled by AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.core import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.ec2 import AWSRetry
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+
+def _describe_alarms(connection, **params):
+    paginator = connection.get_paginator('describe_alarms')
+    return paginator.paginate(**params).build_full_result()
+
+
+def describe_metric_alarms_info(connection, module):
+
+    params = {}
+    params['AlarmNames'] = module.params.get('alarm_names')
+
+    try:
+        describe_metric_alarms_info_response = _describe_alarms(connection, **params)['MetricAlarms']
```

Comment:
What happens if the specified metric_alarm(s) does/do not exist?

Reply:
the code here
```
def _describe_alarms(connection, **params):
    paginator = connection.get_paginator('describe_alarms')
    return paginator.paginate(**params).build_full_result()
```

returns empty dictionary if the alarm does not exit
similar to `{"CompositeAlarms": [], "MetricAlarms": []}`